### ISSUE TITLE
Amesos2:  Tpetra Deprecated function 

### ISSUE THREAD

**Issue by JulienLoiseau** [trilinos/Trilinos#9639]
## Question
@trilinos/Amesos2

I am posting this as a question but it might be a bug. 

The function: `get_crs_helper` is declared _tpetra-deprecated_ here: 
https://github.com/trilinos/Trilinos/blob/240c56b57f731146724178341d1a4f87cf590905/packages/amesos2/src/Amesos2_Util.hpp#L965

It is used in a non _tpetra-deprecated_ function `loadA_impl` here:  
https://github.com/trilinos/Trilinos/blob/240c56b57f731146724178341d1a4f87cf590905/packages/amesos2/src/Amesos2_Superludist_def.hpp#L847

Should the function `loadA_impl` be deprecated or the function `get_crs_helper` be replaced by the new `get_crs_helper_kokkos_view`?

Thank you.

**Comment by cgcgcg** [comment]
@trilinos/amesos2

**Comment by ndellingwood** [comment]
Thanks for reporting, `loadA_impl` should not be deprecated, while `get_crs_helper` should likely be replaced by `get_crs_helper_kokkos_view` as you have determined. I didn't catch this while reviewing some recent code updates, thanks for the catch. If you have changes already made locally please feel free to issue a PR and add me as reviewer, otherwise I can put in a PR for the update.

**Comment by JulienLoiseau** [comment]
I don't have the fix, I just compiled with the deprecated flag for now.

**Comment by etphipp** [comment]
FYI, this causes the build of SPARC against Trilinos-develop to fail, since they build with Tpetra deprecated code disabled, so it would be nice if this could be fixed soon.

**Comment by etphipp** [comment]
And I guess there is apparently no automated testing with Tpetra deprecated code disabled, which there probably should be.

**Comment by rppawlo** [comment]
> And I guess there is apparently no automated testing with Tpetra deprecated code disabled, which there probably should be.

@etphipp - We did have this testing in place in PR testing at one point. When Tpetra deprecated the functions, the corresponding changes were not made in all other trilinos packages, so we had to disable that testing option. The required changes to other packages were not trivial in some cases so it was left to individual packages to decide how to best update. @ccober6 - can we add this to the product leaders agenda?
